src/packages/quickenter/doc.en-US.md

```
@@ -0,0 +1,187 @@
+| Property | Description | Type | Default |
+| --- | --- | --- | --- |
+| title | Option title | `string` | `-` |
+| icon | Option icon | `ReactNode` | `-` |
+| type | Option type | `string` | `-` |
+| url | Jump url | `string` | `-` |
+
+## Theme Customization
+
+### Style Variables
+
+The component provides the following CSS variables for style customization, please refer to [ConfigProvider component](#/en-US/component/configprovider).
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)